pkg/ccl/importccl/import_stmt_test.go

```diff
@@ -656,6 +656,15 @@ func TestImportCSVStmt(t *testing.T) {
 					t.Fatal("expected > 1 SST files")
 				}
 			}
+
+			// Verify spans don't have trailing '/0'.
+			ranges := sqlDB.QueryStr(t, `SHOW testing_ranges FROM TABLE t`)
+			for _, r := range ranges {
+				const end = `/0`
+				if strings.HasSuffix(r[0], end) || strings.HasSuffix(r[1], end) {
```

Comment:
So since `/0` by itself isn’t the problem, but rather having 2x as many ranges, because half were empty, was, I might say we should be testing the number of ranges, not their keys... but this works so I don’t feel strongly.

Reply:
I'm going to keep it as is because I'm not sure there's an easy way to figure out how many ranges there should be, and it works as is.